scripts/lint/check-research-freshness.sh

```
@@ -0,0 +1,130 @@
+
+echo "==> checking research freshness..."
+
+if [[ ! -f "$DATES_FILE" ]]; then
+  echo "  [skip] $DATES_FILE not found"
+  exit 0
+fi
+
+# Portable date diff (works on macOS and Linux)
+days_since() {
+  local date_str="$1"
+  local now today_s date_s
```

Shellcheck (0.11.0)

[warning] now appears unused. Verify use (or export if used externally).

(SC2034)